### DISCUSSION TITLE
A ML knowledge sharing website powered by Kedro

### DISCUSSION THREAD

**Discussion by lhy9381** [kedro-org/kedro#1185] [Idea]
Hello Kedro Community!

As a lead data scientist, after chating with hundreds of data science / machine learning students,  I find it was hard for them to study real industrial ML cases before they land on a job / real-world project, such that very often what they have built from school does not match well with industry needs - companies usually want the new hires have solid technical skills (which can be trained from school) as well as some experience in some specific domain (like the example shown below).

![image](https://user-images.githubusercontent.com/70559000/151586521-77198f55-c82a-45c5-9459-58fcb56b39bf.png)

Last year when I was ponding about this situation, a friend from McKinsey told me about Kedro. I found it fascinating! Comparing to other options (e.g. Jupyter Notebook), it's much easier to host Kedro projects while achiving the objective of domain knowledge sharing. Also, comparing to traditional "data wiki" where we can only read dry documentations, Kedro's pipeline visualization can make the ML flow much easier to understand when it comes to very complex solutions (according to my user research interviews).

I just launched the website prototype recently and wanted to share with the community to collect feedbacks and suggestions, so I can build it better. Big thank you to the Kedro team for creating this amazing framework!

The link - [DasosML.com](http://dasosml.com)

![image](https://user-images.githubusercontent.com/70559000/151586792-a98611df-6395-4834-a14d-ee89748f312b.png)

**Comment by datajoely** [comment]
Awesome work @lhy9381 very cool

**Reply by simkimsia** [reply]
Looks interesting now on phone so cannot check out whole thing I intend to check on desktop tomorrow first thing

thank you @lhy9381

**Comment by simkimsia** [comment]
Is there a repo for this website, @lhy9381 ?

**Reply by simkimsia** [reply]
I have tried to create a new project, but nothing seems to happen @lhy9381 and apologies to the Kedro team if this is not a good place fo rme to ask DASOS ML website

**Reply by lhy9381** [reply]
I just checked the backend and seems there's no projects created as well. In order to create a project, users are required to 1. enter the project title and 2. upload some csv files. Have you done both?

**Reply by simkimsia** [reply]
Oh I didn't upload the csv file 

Do u have a sample I can use?

**Reply by lhy9381** [reply]
sure! the demo project I created before used this dataset [here](https://archive-beta.ics.uci.edu/ml/datasets/bank+marketing)
You can upload them as-is. The website currently supports multi-file uploading but only supports csv format.

**Comment by astrojuanlu** [comment]
Sadly the service doesn't exist anymore and it was never archived by the Wayback Machine... closing.
